portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a todo list. */
+public final class Task {
+
+  private final String firstName;
+  private final String lastName;
+  private final String email;
+  private final String subject;
+
+  public Task(String firstName, String lastName, String email, String subject) {
```

Comment:
A better design here would actually be to make this constructor private

you could create a public static method fromEntity(Entity entity) and then within the method, you could call getProperty on the Entity and then return new Task()